.gitignore

```diff
@@ -20,7 +20,7 @@ tdrs-frontend/.env*.local
 tdrs-backend/vars-backend.yml
 tdrs-backend/coverage.xml
 tdrs-backend/htmlcov/*
-tdrs-backend/.env
+tdrs-backend/.env*
```

Comment:
recently added a `.env.production` file we don't want in the repository

Reply:
This would cause us to ignore changes to `.env.example` - which we do want to commit.

Reply:
Good catch. I have updated accordingly.